web-app/public/locales/es/account.json

```
@@ -0,0 +1,47 @@
+{
+    "title": "Su cuenta de API",
+    "userDetails": "Detalles del usuario",
+    "description": "La API de Mobility Database utiliza la autenticación OAuth2. Para iniciar una solicitud de API exitosa, se debe incluir un token de acceso como token portador en el encabezado HTTP. Los tokens de acceso son válidos por una hora. Para obtener un token de acceso, primero necesitará un token de actualización, que es de larga duración y no caduca.",
+    "support": "Si necesita un token de actualización reemitido o desea que se elimine su cuenta, envíenos un correo electrónico a",
+    "registerApiAnnouncements": "Registrado en Anuncios de API",
+    "refreshToken": {
+       "title": "Token de actualización",
+       "description": "Use su token de actualización para conectarse a la API en su aplicación.",
+       "placeholder": "Su token de actualización está oculto",
+       "yourToken": "Su token de actualización",
+       "toggleVisibility": "Activar o desactivar la visibilidad del token de actualización",
+       "copy": "Copiar token de actualización",
+       "copyToClipboard": "Copiar token de actualización al portapapeles"
+    },
+    "accessToken": {
+       "title": "Cómo generar el token de acceso en su aplicación",
+       "description": {
+          "pt1": "Siga",
+          "cta": "nuestra documentación de OpenAPI",
+          "pt2": "para autenticar su cuenta con el token de actualización".
```

Invalid period at the end of the field value.
```suggestion
          "pt2": "para autenticar su cuenta con el token de actualización."
```